### DISCUSSION TITLE
different version

### DISCUSSION THREAD

**Discussion by mehrdadhaji** [juicedata/juicefs#951] [Q&A] [answered]
hi
whats is diffrent between version 4.5.7 (2021-11-05 c395022) in juicefs website and 0.17.1 (2021-10-28T05:30:05Z 4d11c4b) in github? one of them have format  command one of them not? which version shoud i use?

**Comment by rezaghafkam** [comment]
This is my question too :(

**Comment by xiaogaozi** [comment, marked as answer]
Basically, the hosted service and community version (the GitHub one) have the same core logic, but hosted service has more enterprise related features. You could find more information about JuiceFS Hosted Service [here](https://juicefs.com/docs/en/hosted_service.html).
